window-manager/dwm/config.def.h

```diff
@@ -0,0 +1,17 @@
+/* Set colors for dwm */
+static const char norm_fg[]         = "#0abdc6";
+static const char norm_bg[]         = "#091833";
+static const char norm_border[]     = "#091833";
+static const char sel_fg[]          = "#ea00d9";
+static const char sel_bg[]          = "#091833";
```

Comment:
Are you sure you don't want to use #321959 (or #711c91) for selection background?

Reply:
Yeah, I agree that it looks good with #321959 :laughing: 
Thank you for your feedback!